### ISSUE TITLE
bebop ignores default search engine

### ISSUE THREAD

**Issue by moosotc** [kumabook/bebop#96]
This is an issue given that google is very keyboard unfriendly ("Keyboard navigation for Chrome" for instance does not work)

**Comment by kumabook** [comment]
webextension search api now supported.
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/search/search

I will implement soon

**Comment by kumabook** [comment]
Implemented at https://github.com/kumabook/bebop/pull/133
